src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int lastPosition = 0;
```

Comment:
```suggestion
    private int lastPosition;
```

0 is default value for int